prisma/migrations/20221121232516_unique_masp_weekly/migration.sql

```diff
@@ -0,0 +1,12 @@
+/*
+  Warnings:
+
+  - A unique constraint covering the columns `[asset_name,type,week,main]` on the table `masp_transactions` will be added. If there are existing duplicate values, this will fail.
+  - Added the required column `week` to the `masp_transactions` table without a default value. This is not possible if the table is not empty.
+
+*/
+-- AlterTable
+ALTER TABLE "masp_transactions" ADD COLUMN     "week" INTEGER NOT NULL;
```

Comment:
Wondering how this will be handled on deploy to staging/production, if rows exist it should fail, no?